server/models/index.js

```diff
@@ -1,7 +1,9 @@
 // Export all Models
 
 var User = require("./User");
+var Order = require("./order");
```

Comment:
Pls be consistent with the name ... it should be `Order` not `order` @RubyTamrakar